engineering_tools/opendrive_lanelet2_converter/opendrive2lanelet2convertor.py

```diff
@@ -0,0 +1,169 @@
+import os
```

Comment:
Add the apache2 license found in more of our files to the top of this document

Reply:
Also add the shebang to the top of the file for the correct python version this will work with
```
#!/usr/bin/env python3
```
or
```
#!/usr/bin/env python2
```